__tests__/app.test.js

```diff
@@ -8,16 +8,15 @@ afterAll(() => db.end());
 
 beforeEach(() => seed(data));
 
-describe('GET /api/bananas', () => {
-    test('404: should respond with "bad request" if the endpoint does not exist', () => {
-        return request(app)
-        .get('/api/bananas')
-        .expect(404)
-        .then(({body}) => {
-            console.log(body)
-            expect(body.msg).toEqual('Bad Request')
-        })
-    })
+describe("GET /api/bananas", () => {
+  test('404: should respond with "bad request" if the endpoint does not exist', () => {
+    return request(app)
+      .get("/api/bananas")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toEqual("Bad Request");
```

Comment:
This would be 404 - route not found - its not a bad request as it technically is valid, it just doesnt exist.